router/pods_test.go

```diff
@@ -800,30 +800,26 @@ func TestGetHealthCheckFromPodPort(t *testing.T) {
 
 // TestValidatePath tests the internal function to validate a proper path used in annotations
 func TestValidatePath(t *testing.T) {
-	testNoPrefix := "test"
-	testPathFail := "/test/%2a/%"
-	testPathPass := "/test/%2a/aa/a"
-	testPathPassProperEncoding := "/test/par%2ate/aa/a"
-	testPathFailInvalidEncoding := "/test/hello%zzworld/aa/a"
```

Comment:
Ah, I get what you're saying now.  :)  If the user provides a regex, you cannot use the URI validation regex to validate that.  We're on the same page...although I wonder if there's a better way.  What about different values?  `path` vs. `pathRegex`?  Just thinking aloud.